content/docs/user-guide/editor/viewport/advanced-operations.md

```diff
@@ -0,0 +1,71 @@
+---
+linkTitle: Advanced Operations
+title: Advanced 3D Viewport Operations
+description: Learn the advanced operations to manipulate entities in the 3D Viewport in Open 3D Engine (O3DE)
+weight: 400
+---
+
+## Reset an entity's transform, rotation, or scale
+
+You can reset the transform for an entity, so that you return the entity to a default location, rotation, and scale.
+
+In the viewport, select an entity or group of entities and press **R**.
+
+Note that the effect of the reset depends on the current manipulator mode. For example, if you are in the translation mode, the translation is reset for the selected entity or entities.  To reset the selected entity's scale, switch to the scale manipulator first.
+
+![Modify entities and reset their transform in O3DE.](/images/user-guide/viewportinteractionmodel/viewport-selection-model-reset-transform-1.gif)
+
+## Modifying manipulators
+
+You can set manipulators to have a custom location, rotation, or scale.  
+
+1. In the viewport, select an entity.
+
+1. Select the transform mode of the manipulator you want to modify.
+
+1. Press and hold **Ctrl**, and **left-click and drag** the manipulator to modify it's location, rotation, or scale.
+
+In the following example, the manipulator is moved away from the car, switches to rotate mode, and rotates the car.
+
+![Move and rotate an entity using a manipulator in O3DE.](/images/user-guide/viewportinteractionmodel/viewport-selection-model-3.gif)
+
+## Reset a manipulator's transform, rotation, or scale
+
+You can reset a manipulator to it's default location, rotation, or scale by pressing **Ctrl + R** while the related manipulator is active.
```

Comment:
Nit: same as above (it's vs its). I will leave any others i find so might be worth ctl-f'ing for any other instances.

Reply:
Lol, embarrassing, but thank you!